bugbot/rules/not_landed.py

```diff
@@ -168,12 +168,13 @@ def handle_attachment(self, attachment, res):
     def get_patch_data(self, bugs):
         """Get patch information in bugs"""
         nightly_pat = Bugzilla.get_landing_patterns(channels=["nightly"])[0][0]
+        autoland_pat = Bugzilla.get_landing_patterns(channels=["autoland"])[0][0]
```

Comment:
nit: you could run a single `Bugzilla.get_landing_patterns` once with `channels=["autoland", "nightly"]`

Reply:
then store the patterns in a list, and use them with `any` at line 177